types/sass-true/index.d.ts

```diff
@@ -0,0 +1,37 @@
+// Type definitions for sass-true 6.0
+// Project: https://github.com/oddbird/true
+// Definitions by: robertmaier <https://github.com/robertmaier>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+import type { Options } from 'sass';
+
+export interface TrueOptions {
+    sass?: typeof import('sass');
+    describe: (description: string) => void;
+    it: (description: string) => void;
+}
+
+export interface Assertion {
+    description: string;
+    output: string;
+    assertionType: string;
+    expected: string;
+    details?: string;
+}
+
+export interface Module {
```

Comment:
Where are `Module` and `Test` used?

Reply:
`Module` is used as the return type of the `parse` function. Test is used within the `Module` type

Reply:
🤦‍♂️